frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Card, Table } from "react-bootstrap";
+import { ImplementationMetadata } from "../../data/parseReportData";
+
+const DialectCompliance: React.FC<{ specificData: ImplementationMetadata }> = ({
+  specificData,
+}) => {
+  const dialectMapping: { [key: string]: string } = {
+    "draft2020-12": "https://json-schema.org/draft/2020-12",
+    "draft2019-09": "https://json-schema.org/draft/2019-09",
+    draft7: "http://json-schema.org/draft-07",
+    draft6: "http://json-schema.org/draft-06",
+    draft4: "http://json-schema.org/draft-04",
+    draft3: "http://json-schema.org/draft-03",
+  };
+
+  return (
+    <Card className="mx-auto mb-3 w-75">
+      <Card.Header>Compliance</Card.Header>
+      <Card.Body>
+        <Table className="table-hover sm">
+          <thead>
+            <tr>
+              <th rowSpan={2} scope="col" className="text-center align-middle">
+                Supported Dialects
+              </th>
+              <th colSpan={3} className="text-center">
+                Tests
+              </th>
+            </tr>
+            <tr className="text-center">
+              <th>Unsuccessful</th>
```

Comment:
The terminology here is confusing to me (though it's undoubtedly confusing to everyone, and will require some tooltips to explain).

But in the backend, the term "unsuccessful" is used to refer to tests which *either* fail (i.e. have a result different than the expected one) or error (i.e. the test caused an exception in the harness).

That's distinct from "failures" which are just the former.

Which does this mean here? Combined or just failures?

Reply:
It should be `Failed`, actually. (Both **failed** and **errored** tests are displayed separately here)